wire-ios/Wire-iOS/Sources/UserInterface/Availability/AvailabilityTitleView.swift

```diff
@@ -98,24 +102,44 @@ final class AvailabilityTitleView: TitleView, ZMUserObserver {
     /// Refreshes the content and appearance of the view.
     func updateConfiguration() {
         updateAppearance()
-        updateContent()
+        Task {
+            await updateContent()
+        }
     }
 
     /// Refreshes the content of the view, based on the user data and the options.
-    private func updateContent() {
+    private func updateContent() async {
         typealias AvailabilityStatusStrings = L10n.Accessibility.AccountPage.AvailabilityStatus
 
         let availability = user.availability
         let fontStyle: FontSize = options.contains(.useLargeFont) ? .normal : .small
-        let icon = AvailabilityStringBuilder.icon(
-            for: availability,
-               with: AvailabilityStringBuilder.color(for: availability),
-               and: fontStyle)
+        var icons = [
+            AvailabilityStringBuilder.icon(
+                for: availability,
+                with: AvailabilityStringBuilder.color(for: availability),
+                and: fontStyle
+            )
+        ]
         let isInteractive = options.contains(.allowSettingStatus)
         var title = ""
 
         if options.contains(.displayUserName) {
             title = user.name ?? ""
+            do {
+                let verificationStatuses = try await getSelfUserVerificationStatusUseCase.invoke()
+                if verificationStatuses.isProteusVerified {
+                    let attachment = NSTextAttachment(image: Asset.Images.verifiedShield.image)
+                    attachment.bounds = .init(origin: .init(x: 0, y: -2), size: attachment.image!.size)
+                    icons.insert(attachment, at: 0)
+                }
+                if verificationStatuses.isMLSCertified {
+                    let attachment = NSTextAttachment(image: Asset.Images.certificateValid.image)
+                    attachment.bounds = .init(origin: .init(x: 0, y: -2), size: attachment.image!.size)
+                    icons.insert(attachment, at: 0)
```

Comment:
Why do you insert in the specific position? It's the same position as a "blue shield" (isProteusVerified), it will override it.

Reply:
It's not replacing the first item, but prepending the list.